source/connect/tls-ssl-connection.txt

```diff
@@ -11,3 +11,106 @@ TLS/SSL Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`TLS / SSL` tab allows you to connect deployments via TLS / SSL.
```

Comment:
We should replace "via" with "using". "Via" can be difficult for non-English speakers.

See https://docs.mongodb.com/meta/style-guide/terminology/general-term-guidelines/avoid-obscure-words/.

Reply:
Done.